mainnet/13-wysiwyg-finance-transfer/evidence/against.md

```diff
@@ -13,3 +13,4 @@ Given guideline 3.2 of the Aragon Network Cash Agreement:
 - Proposals MUST be accompanied by an accurate American English-language Justification of the proposal.
 
 This proposal should be ruled "BLOCK" by Aragon Court.
+
```

Comment:
This is not hugely important, but it may be useful to control the line breaks via git: https://docs.github.com/en/github/using-git/configuring-git-to-handle-line-endings#per-repository-settings.

These line breaks _will_ modify the resulting CID, so it is best to be standardized across all users.

Reply:
Yup, it was on purpose, I wanted to fool the pinnet to force uploading a separate file